index.html

```diff
@@ -1,16 +1,65 @@
 <!DOCTYPE html>
 <html lang="en">
 <head>
-    <meta charset="UTF-8">
-    <meta http-equiv="X-UA-Compatible" content="IE=edge">
-    <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Weather Report</title>
-    <link rel="preconnect" href="https://fonts.gstatic.com">
-    <link href="https://fonts.googleapis.com/css2?family=Rubik&display=swap" rel="stylesheet">
-    <link rel="stylesheet" href="styles/index.css" />
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Yulia & Hannah's Weather Report</title>
+  <link rel="preconnect" href="https://fonts.gstatic.com">
+  <link href="https://fonts.googleapis.com/css2?family=Rubik&display=swap" rel="stylesheet">
+  <link rel="stylesheet" href="styles/index.css" />
+  <link rel="preconnect" href="https://fonts.googleapis.com">
+  <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+  <link href="https://fonts.googleapis.com/css2?family=Noto+Serif&family=Pacifico&family=Roboto:wght@400;700&family=Tsukimi+Rounded&display=swap" rel="stylesheet">
 </head>
 <body>
-    <script src="./src/index.js"></script>
-    <script src="./node_modules/axios/dist/axios.min.js"></script>
+  <header>
+      <h1>Weather Report</h1>
+      <h2>Of glorious city ✨ <span id="city"></span> ✨</h2>
+      <h2 id="date-time"></h2>
+  </header>
+  <main>
+    <section id="temp-section" class="temp-container">
+      <h2 id="temp-section-title" class="temp-section-title">Temperature</h2>
+      <section class="temp-interface-container">
+        <section class="up-down-container">
+          <button id="up-temp" class="up-button">&#8657;</button>
+          <h2 id="current-temp" class="current-temp"></h2>
+          <button id="down-temp" class="down-button">&#8659;</button>
+        </section>
+        <button id="real-time-button" class="real-time-button">Get Real Time Temperature</button>
+      </section>
+    </section>
+    <section id="sky-section" class="sky-container">
+      <label for="sky-select">Sky
+        <select id="sky-select" class="sky-selector" name="skies">
+          <option value="default">Please choose an option</option>
+          <option value="sunny">Sunny</option>
+          <option value="cloudy">Cloudy</option>
+          <option value="rainy">Rainy</option>
+          <option value="snowy">Snowy</option>
+        </select>
+      </label>
```

Comment:
Exceptional semantic HTML all around! Great job remembering to give our `option` tags a `value` field so it's accessible to screen readers!